FLEDGE.md

```diff
@@ -233,6 +235,10 @@ Buyers may want to make on-device decisions that take into account real-time dat
 
 The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trusted_bidding_signals_url`, the hostname of the top-level webpage where the ad will appear `publisher.com` is provided by the browser, and `keys` is a list of `trusted_bidding_signals_keys` strings, perhaps coalesced (for efficiency) across any number of interest groups that share a `trusted_bidding_signals_url`.  The response from the server should be a JSON object whose keys are key1, key2, etc., and whose values will be made available to the buyer's bidding functions (un-coalesced).
 
+
+Similarly, sellers may want to fetch information about a specific creative, e.g. the results of some out-of-band ad scanning system.  This works in the same way, with the base URL coming from the `trusted_scoring_signals_url` property of the seller's auction configuration object, and the keys being the `render_url` fields of all the ads in the `ads` fields of all interest groups in the auction.  The value associated with a `render_url` key is provided as the `trusted_scoring_signals` parameter to the seller's `score_ad()` function.
+
+
 _As a temporary mechanism_ during the First Experiment timeframe, the buyer can fetch these bidding signals from any server, including one operated by the buyer themselves (a "Bring Your Own Server" model).  However, in the final version after the removal of third-party cookies, the request will only be sent to a trusted key-value-type server.  Because the server is trusted, there is no k-anonymity constraint on this request.  The browser needs to trust that the server's return value for each key will be based only on that key and the hostname, and that the server does no event-level logging and has no other side effects based on these requests. 
```

Comment:
Should we revise "the buyer can fetch" to also mention sellers?

Reply:
Done